tensorflow/python/ops/image_ops_impl.py

```diff
@@ -789,11 +789,9 @@ def pad_to_bounding_box(image, offset_height, offset_width, target_height,
     `[target_height, target_width, channels]`
 
 Usage Example:
-    '''python
-    import tensorflow as tf
-    x = tf.random.normal(shape=(256, 256, 3))
-    x = tf.image.pad_to_bounding_box(x, 2, 2, 260, 260)
-    '''
+>>> x = tf.random.normal(shape=(256, 256, 3))
+>>> x = tf.image.pad_to_bounding_box(x, 2, 2, 260, 260)
+<x.shape = TensorShape([Dimension(260), Dimension(260), Dimension(3)])>
```

Comment:
It should read TensorShape([260, 260, 3]); did you run the example with tf1?

Reply:
That seemed to be the problem. I have fixed the issue.